Library/Formula/sonar.rb

```diff
@@ -46,4 +44,8 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    assert_match /SonarQube/, pipe_output("#{bin}/sonar status")
```

Comment:
Apologies, this came from me. ` sonar status ` doesn't seem to exit by itself, or at least not for a prolonged period, so I didn't know if that would impact ` shell_output ` in the same manner.